### PR TITLE
Add SessionTracker to state and fix cookie issue

### PR DESCRIPTION
**Pull request by tomsoir**
https://reddit.atlassian.net/browse/GROW-727
P.S. XPromo on loading has been tested also.

### CODE REVIEW

**Comment by tomsoir** [comment]
👓

File: src/app/reducers/sessionTracker.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    // so this ensures the api receives the most up-to-date value.
+    case accountActions.RECEIVED_ACCOUNT: {
+      const cookieHeadersPath = ['apiResponse','meta','set-cookie'];
+      const setCookieHeaders = get(action, cookieHeadersPath, []);
+      let sessionTrackerState = pick(state, ['id', 'value']); // prev. value
+
+      if (setCookieHeaders.length) {
+        setCookieHeaders.forEach(setCookieHeader => {
+          const { name, value } = setCookieParser.parse(setCookieHeader)[0];
+          if (name === 'session_tracker') {
```
Comment (schwers) [inline src/app/reducers/sessionTracker.js]: Small nitpick, please use the constant `SESSION_TRACKER` from 'lib/getSessionIDFromCookie`

Comment (tomsoir) [inline src/app/reducers/sessionTracker.js]: 💇‍♂️

**Comment by tomsoir** [comment]
💇‍♂️

**Pull request merged**